apps/omg_watcher/config/test.exs

```diff
@@ -6,12 +6,11 @@ config :omg_watcher, OMG.Watcher.Web.Endpoint,
   http: [port: 4001],
   server: false
 
-# Configure your database
 config :omg_watcher, OMG.Watcher.DB.Repo,
-  adapter: Sqlite.Ecto2,
-  database: "/tmp/omisego/ecto_simple_" <> Integer.to_string(:rand.uniform(10_000_000)) <> ".sqlite3"
-
-config :omg_watcher, OMG.Watcher.DB.Repo, pool: Ecto.Adapters.SQL.Sandbox
+  adapter: Ecto.Adapters.Postgres,
+  pool: Ecto.Adapters.SQL.Sandbox,
+  # DATABASE_URL format is following `postgres://{user_name}:{password}@localhost/{database_name}`
```

Comment:
I think it's `postgres://{user_name}:{password}@{machine}:{port}/{database_name}`

Reply:
Yes, thanks!